diff --git a/_pieces_lib/urllib3/contrib/emscripten/__init__.py b/_pieces_lib/urllib3/contrib/emscripten/__init__.py
index 8a3c5be..5beb505 100644
--- a/_pieces_lib/urllib3/contrib/emscripten/__init__.py
+++ b/_pieces_lib/urllib3/contrib/emscripten/__init__.py
@@ -1,6 +1,6 @@
 from __future__ import annotations

-import urllib3.connection
+import Pieces._pieces_lib.urllib3.connection

 from ...connectionpool import HTTPConnectionPool, HTTPSConnectionPool
 from .connection import EmscriptenHTTPConnection, EmscriptenHTTPSConnection
@@ -12,5 +12,5 @@ def inject_into_urllib3() -> None:
     # if it isn't ignored
     HTTPConnectionPool.ConnectionCls = EmscriptenHTTPConnection
     HTTPSConnectionPool.ConnectionCls = EmscriptenHTTPSConnection
-    urllib3.connection.HTTPConnection = EmscriptenHTTPConnection  # type: ignore[misc,assignment]
-    urllib3.connection.HTTPSConnection = EmscriptenHTTPSConnection  # type: ignore[misc,assignment]
+    Pieces._pieces_lib.urllib3.connection.HTTPConnection = EmscriptenHTTPConnection  # type: ignore[misc,assignment]
+    Pieces._pieces_lib.urllib3.connection.HTTPSConnection = EmscriptenHTTPSConnection  # type: ignore[misc,assignment]
diff --git a/_pieces_lib/urllib3/contrib/pyopenssl.py b/_pieces_lib/urllib3/contrib/pyopenssl.py
index b89a6da..8a1e544 100644
--- a/_pieces_lib/urllib3/contrib/pyopenssl.py
+++ b/_pieces_lib/urllib3/contrib/pyopenssl.py
@@ -28,7 +28,7 @@ like this:
 .. code-block:: python

     try:
-        import urllib3.contrib.pyopenssl
+        import Pieces._pieces_lib.urllib3.contrib.pyopenssl
         urllib3.contrib.pyopenssl.inject_into_urllib3()
     except ImportError:
         pass
diff --git a/_pieces_lib/urllib3/http2.py b/_pieces_lib/urllib3/http2.py
index 15fa9d9..b45bb99 100644
--- a/_pieces_lib/urllib3/http2.py
+++ b/_pieces_lib/urllib3/http2.py
@@ -8,8 +8,8 @@ import h2.config  # type: ignore[import-untyped]
 import h2.connection  # type: ignore[import-untyped]
 import h2.events  # type: ignore[import-untyped]

-import urllib3.connection
-import urllib3.util.ssl_
+import Pieces._pieces_lib.urllib3.connection
+import Pieces._pieces_lib.urllib3.util.ssl_
 from urllib3.response import BaseHTTPResponse

 from ._collections import HTTPHeaderDict
@@ -216,14 +216,14 @@ class HTTP2Response(BaseHTTPResponse):

 def inject_into_urllib3() -> None:
     HTTPSConnectionPool.ConnectionCls = HTTP2Connection
-    urllib3.connection.HTTPSConnection = HTTP2Connection  # type: ignore[misc]
+    Pieces._pieces_lib.urllib3.connection.HTTPSConnection = HTTP2Connection  # type: ignore[misc]

     # TODO: Offer 'http/1.1' as well, but for testing purposes this is handy.
-    urllib3.util.ssl_.ALPN_PROTOCOLS = ["h2"]
+    Pieces._pieces_lib.urllib3.util.ssl_.ALPN_PROTOCOLS = ["h2"]


 def extract_from_urllib3() -> None:
     HTTPSConnectionPool.ConnectionCls = orig_HTTPSConnection
-    urllib3.connection.HTTPSConnection = orig_HTTPSConnection  # type: ignore[misc]
+    Pieces._pieces_lib.urllib3.connection.HTTPSConnection = orig_HTTPSConnection  # type: ignore[misc]

-    urllib3.util.ssl_.ALPN_PROTOCOLS = ["http/1.1"]
+    Pieces._pieces_lib.urllib3.util.ssl_.ALPN_PROTOCOLS = ["http/1.1"]
